lib/utils.ts

```diff
@@ -170,6 +177,13 @@ export const CHARITY_MAP: Record<string, any> = {
       "Can't decide? Follow group wisdom. Fund the 50+ most popular charities on Endaoment",
     type: "",
   },
+  ["REFUGEE_CRISIS"]: {
```

Comment:
https://github.com/Glo-Foundation/glo-wallet/pull/409 can you merge this one into yours @Riglanto ?